vlmeval/dataset/utils/judge_util.py

```diff
@@ -17,7 +17,7 @@ def build_judge(**kwargs):
             'gpt-4-0125': 'gpt-4-0125-preview',
             'gpt-4-0409': 'gpt-4-turbo-2024-04-09',
             'chatgpt-1106': 'gpt-3.5-turbo-1106',
-            'chatgpt-0125': 'gpt-3.5-turbo-0125',
+            'chatgpt-0125': 'gpt-3.5-turbo', #'gpt-3.5-turbo-0125',
```

Comment:
It's preferred to set the specific version clearly, do not use `gpt-3.5-turbo` (we don't know the version).